client/webserver/live_test.go

```diff
@@ -366,6 +354,16 @@ var tExchanges = map[string]*core.Exchange{
 				Confs: 10,
 				Amt:   1e11,
 			},
+			"mona": {
+				ID:    22,
+				Confs: 5,
+				Amt:   1e10,
+			},
+			"vtc": {
+				ID:    28,
+				Confs: 5,
+				Amt:   1e11,
+			},
```

Comment:
@chappjc, sorry I was not too clear with my comments. Creating a `mona` test wallet will panic here when trying to pay reg fees cause it was not previously available. https://github.com/decred/dcrdex/blob/178daeb258c377a4c9318a41e7be7ae7e6f7fbba/client/webserver/live_test.go#L1408